src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php

```diff
@@ -45,6 +45,7 @@ class UnusedTagsPass implements CompilerPassInterface
         'container.service_subscriber',
         'container.stack',
         'controller.argument_value_resolver',
+        'controller.pinned_argument_value_resolver',
```

Comment:
we don't need this tag anymore to me

Reply:
What do you mean? This tag is used to populate `ArgumentResolver::$namedResolvers` :thinking:

Reply:
we can use controller.argument_value_resolver to compute this

Reply:
Wouldn’t that mean that every pinnable resolver will also be present in `ArgumentResolver::$argumentValueResolvers`? I thought you wanted a way to opt-out from that.